gox/plan.sh

```diff
@@ -0,0 +1,24 @@
+pkg_origin=core
+pkg_name=gox
+pkg_version=master
```

Comment:
Yep, it seems @mitchellh hasn't released a tagged version in a while. And to build `vault` requires `-tags` which is only available via master. 😢